code_generator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -108,7 +108,10 @@ namespace <#= qualifiedNamespace #>
                 Unity.Entities.Entity entity;
                 if (!view.TryGetEntity(op.EntityId.Id, out entity))
                 {
-                    Debug.LogErrorFormat(TranslationErrors.OpReceivedButNoEntity, op.GetType().Name, op.EntityId.Id);
+                    view.LogDispatcher.HandleLog(new LogEvent("Entity not found.", LogType.Error)
```

Comment:
Entity not found doesn't really give enough information. It should say in what context it was looking.